R/create_demo_templates.R

```diff
@@ -0,0 +1,38 @@
+#' @title Create Demo Templates
+#' @description saves .png files of each template using synthetic data
+library(readr)
+library(dplyr)
+library(ggplot2)
```

Comment:
Since the functions in this file are getting included in the package, the libraries required should be annotated and let the R package machinery load them.


```
#' @importFrom readr read_csv
#' @importFrom ggplot2 ggsave
create_template <- function(name, recipient) {
```